app/templates/package.json

```diff
@@ -27,10 +27,11 @@
   },
   "scripts": {
     "prepublish": "npm run compile",
-    "publish": "git push origin && git push origin --tags",
+    "publish": "npm run changelog && git push origin && git push origin --tags",
     "release:patch": "npm version patch && npm publish",
     "release:minor": "npm version minor && npm publish",
     "release:major": "npm version major && npm publish",
+    "changelog": "github_changelog_generator && git add CHANGELOG.md && git commit -am \"Updating changelog\"",
```

Comment:
Where does `github_changelog_generator` come from? Assuming it is a script somewhere but not seeing it.